src/components/character-aggregation-display.tsx

```diff
@@ -48,9 +46,8 @@ export default function CharacterAggregationDisplay({
     )
     .sort((a, b) => b.value - a.value);
 
-  const Row = ({ index, key, style }) => (
+  const Row = ({ index, style }) => (
```

Comment:
I think this does need the key, it's iterated over by the the `react-window` lib to create the rows in the list.